integration/kubernetes/helm/alluxio/templates/alluxio-worker.yaml

```diff
@@ -30,107 +30,107 @@ spec:
-              readOnly: true
-            {{- end }}
-            {{- end }}
-            {{- end }}
-        - name: alluxio-job-worker
-          image: {{ .Values.image }}:{{ .Values.imageTag }}
-          imagePullPolicy: {{ .Values.imagePullPolicy }}
-          resources:
-            limits:
-              cpu: {{ .Values.resources.jobWorker.limits.cpu }}
-              memory: {{ .Values.resources.jobWorker.limits.memory }}
-            requests:
-              cpu: {{ .Values.resources.jobWorker.requests.cpu }}
-              memory: {{ .Values.resources.jobWorker.requests.memory }}
-          command: ["/entrypoint.sh"]
-          args: ["job-worker"]
-          env:
-          - name: ALLUXIO_WORKER_HOSTNAME
-            valueFrom:
-              fieldRef:
-                fieldPath: status.hostIP
-          - name: ALLUXIO_JOB_WORKER_JAVA_OPTS
-            value: " -Dalluxio.worker.hostname=$(ALLUXIO_WORKER_HOSTNAME) "
-          envFrom:
-          - configMapRef:
-              name: alluxio-config
-          ports:
-          - containerPort: {{ .Values.resources.jobWorker.ports.rpc }}
-            name: job-rpc
-          - containerPort: {{ .Values.resources.jobWorker.ports.data }}
-            name: job-data
-          - containerPort: {{ .Values.resources.jobWorker.ports.web }}
-            name: job-web
-          volumeMounts:
-            - name: alluxio-ramdisk
-              mountPath: /dev/shm
-            - name: alluxio-domain
-              mountPath: /opt/domain
-            {{- if .Values.secrets }}
-            {{- if .Values.secrets.worker }}
-            {{- range $key, $val := .Values.secrets.worker }}
-            - name: secret-{{ $key }}-volume
-              mountPath: /secrets/{{ $val }}
-              readOnly: true
-            {{- end }}
-            {{- end }}
-            {{- end }}
-      restartPolicy: Always
-      volumes:
+      - name: alluxio-worker
+        image: {{ .Values.image }}:{{ .Values.imageTag }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}
+        resources:
+          limits:
+            cpu: {{ .Values.resources.worker.limits.cpu }}
+            memory: {{ .Values.resources.worker.limits.memory }}
+          requests:
+            cpu: {{ .Values.resources.worker.requests.cpu }}
+            memory: {{ .Values.resources.worker.requests.memory }}
+        command: ["/entrypoint.sh"]
+        args: ["worker-only", "--no-format"]
+        env:
+        - name: ALLUXIO_WORKER_HOSTNAME
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
+        - name: ALLUXIO_WORKER_JAVA_OPTS
+          value: {{ print " -Dalluxio.worker.hostname=$(ALLUXIO_WORKER_HOSTNAME) " | quote }}
+        envFrom:
+        - configMapRef:
+            name: {{ .Values.const.configName }}
+        ports:
+        - containerPort: {{ .Values.resources.worker.ports.rpc }}
+          name: rpc
+        - containerPort: {{ .Values.resources.worker.ports.web }}
```

Comment:
@madanadit Just confirming, worker does not need 19200 prepared for embedded journal.

Reply:
correct